lib/ui/view/forgot_password/forgot_password_otp/forgot_password_otpviewmodel.dart

```diff
@@ -1,5 +1,4 @@
 import 'dart:async';
-
 import 'package:flutter/material.dart';
```

Comment:
material import in viewmodel not consistent with stacked, please remove. The stacked way is using [FormViewModel](https://www.filledstacks.com/post/building-flutter-login-and-sign-up-forms/) or check our repo [here](https://github.com/zurichat/zc_app/tree/dev/lib/ui/view/otp) for sample implementation

Reply:
i didn't work on that page. i only pulled from the dev branch

Reply:
okay

Reply:
done